File: c/Factorial.c
#include<stdio.h>

int factorial(int number)
{
    if(number == 0)
        return 1;
    else
        return(number*factorial(number - 1));
}
int main()
{
    int n, num;
    printf("Enter no. to find factorial: ");
    scanf("%d", &n);
    if (n>0)
        num = factorial(n);
    else
        printf("Please enter a positive integer number");
    printf("%d! = %d", n, num);
}

File: Java/Factorial.java
public class Factorial {
    public static void main(String[] args) {
        int num = 5;
        int result = 1;

        for(int i = 1; i <= num; i++){
            result *= i;
        }

        System.out.println("Factorial of " + num + " is equal to " + result);
    }
}
